eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
-    <!-- Manually updated version from 6.0.0 to address CVE-2021-43877 -->
+    <!-- Manually updated version from 6.0.0 to address CVE-2023-29331 -->
```

Comment:
8.0?

Reply:
Not sure how to interpret your comment, @BrennanConroy, sorry...

Reply:
Update the comment to say 8.0, not 6.0

Reply:
It says "from 6.0" not to?